src/base/Constants.hh

```diff
@@ -31,29 +50,33 @@ constexpr real_type pi = 3.14159265358979323846; // truncated
 
 //@{
 //! Physical constants with *exact* values as defined by SI
-constexpr real_type c_light    = 299792458.0 * units::meter / units::second;
-constexpr real_type h_planck   = 6.62607015e-34 * units::joule * units::second;
-constexpr real_type e_electron = 1.602176634e-19 * units::coulomb;
+constexpr real_type c_light      = 299792458. * units::meter / units::second;
+constexpr real_type h_planck     = 6.62607015e-34 * units::joule * units::second;
+constexpr real_type e_electron   = 1.602176634e-19 * units::coulomb;
 constexpr real_type k_boltzmann  = 1.380649e-23 * units::joule / units::kelvin;
 constexpr real_type na_avogadro  = 6.02214076e23; // [1/mol]
 constexpr real_type kcd_luminous = 683;           // lm/W
 //@}
 
 //@{
-//! Experimental physical constants
-constexpr real_type alpha_fine_structure = 1 / 137.035999084; // CODATA 2018
-constexpr real_type atomic_mass = 12. * units::gram / na_avogadro; // SI 2019
+//! Exact derivative constants
+constexpr real_type hbar_planck = h_planck / (2 * pi);
 //@}
 
 //@{
-//! Derived physical constants
-constexpr real_type eps_electric
-    = e_electron * e_electron
-      / (2 * alpha_fine_structure * h_planck * c_light); // vacuum permittivity
-constexpr real_type mu_magnetic
-    = 1 / (eps_electric * c_light * c_light); // vaccum permeability
+//! Experimental physical constants from CODATA 2018.
+constexpr real_type a0_bohr              = 5.29177210903e-11 * units::meter; //!< Bohr radius
+constexpr real_type alpha_fine_structure = 7.2973525693e-3;
```

Comment:
Any reason on why we're not setting `alpha = e_electron^2 / (4*pi*eps_electric*hbar*c_light)`?
Same for other derived quantities, such as `a0_bohr`?

Reply:
Yeah did you read the comment I wrote above in the namespace documentation? I think writing out the numerical values improves clarity, and I check the equivalent definitions in the unit tests.

Reply:
Ok, that's fair. Should have paid more attention to the comment.